### PR TITLE
updating iogi

### LINKED ISSUE #1007: update IOGI version before next major release
a new version of iogi with some small fixes will be released soon

### PR DESCRIPTION
**Pull request by marcosalles**
Closes #1007

### CODE REVIEW

**Comment by marcosalles** [comment]
tested with music jungle and looks like it's working

File: vraptor-blank-project/pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 	<groupId>br.com.caelum</groupId>
 	<artifactId>vraptor-blank-project</artifactId>
-	<version>4.2.0-RC3-SNAPSHOT</version>
+	<version>4.2.0-RC4-SNAPSHOT</version>
```
Comment (Turini) [inline vraptor-blank-project/pom.xml]: nice catch! thks

**Comment by Turini** [comment]
@marcosalles thanks for your pr, that's great!
travis is failing, but it's not about your changes. I'll take a look on that

**Comment by Turini** [comment]
I did fetch and test this change and everything is working fine.
that sounds like a travis bug... so I'm merging here. Thanks again @marcosalles

**Pull request merged**